packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -238,6 +241,9 @@ class AndroidSdkVersion implements Comparable<AndroidSdkVersion> {
     if (_canRun(zipalignPath) != null)
       return <String>[_canRun(zipalignPath)];
 
+    if (_canRun(apksignerPath) != null)
+      return <String>[_canRun(apksignerPath) + ' (Requires Android SDK Build Tools 24.0.3 or newer.)'];
```

Comment:
The version number of build tools is printed just above this message in `flutter doctor`. I added a link to the installation instructions to the message `flutter doctor` prints below this error.